.eslintrc.js

```diff
@@ -14,96 +14,27 @@
-    env: {
-        browser: true,
-        es6: true,
-        jest: true,
-        node: true
-    },
-    extends: [
-        "plugin:cypress/recommended",
-        "eslint:recommended",
-        "plugin:import/typescript"
-    ],
-    overrides: [
-        {
-            env: {
-                browser: true,
-                es6: true,
-                node: true
-            },
-            extends: [
-                "eslint:recommended",
-                "plugin:@typescript-eslint/eslint-recommended",
-                "plugin:@typescript-eslint/recommended"
-            ],
-            files: ["**/*.tsx", "**/*.ts"],
-            parser: "@typescript-eslint/parser",
-            parserOptions: {
-                ecmaVersion: 9,
-                sourceType: "module"
-            },
-            rules: {
-                "@typescript-eslint/explicit-function-return-type": 0,
-                "@typescript-eslint/no-empty-function": [ "error", { "allow": ["constructors"] } ],
-                "@typescript-eslint/no-explicit-any": 0,
-                "@typescript-eslint/no-inferrable-types": "off",
-                "@typescript-eslint/no-use-before-define": [
-                    "warn",
-                    {
-                        classes: false,
-                        functions: false,
-                        typedefs: false,
-                        variables: false
-                    }
-                ],
-                "@typescript-eslint/triple-slash-reference": 0,
-                "eol-last": "error",
-                "no-use-before-define": "off"
-            }
-        }
-    ],
-    parserOptions: {
-        ecmaVersion: 9,
-        sourceType: "module"
-    },
-    plugins: ["import"],
-    rules: {
-        "comma-dangle": ["warn", "never"],
-        "cypress/no-unnecessary-waiting": ["off"],
-        "eol-last": "error",
-        "import/order": [
-            "warn",
-            {
-                "alphabetize": {
-                    caseInsensitive: true,
-                    order: "asc"
-                },
-                "groups": ["builtin", "external", "index", "sibling", "parent", "internal"]
-            }
-        ],
-        "max-len": ["warn", { "code": 120 }],
-        "no-console": "warn",
-        "no-duplicate-imports": "warn",
-        "object-curly-spacing": ["warn", "always"],
-        "quotes": ["warn", "double"],
-        "sort-imports": ["warn", {
-            "ignoreCase": false,
-            "ignoreDeclarationSort": true,
-            "ignoreMemberSort": false
-        }],
-        "sort-keys": [
-            "warn",
-            "asc",
-            {
-                "caseSensitive": true,
-                "minKeys": 2,
-                "natural": false
-            }
-        ]
-    }
+  env: {
+    // other environments
+    jest: true,
+  },
+  extends: [
+    "plugin:@wso2/typescript",
+    "plugin:@wso2/strict",
+    "plugin:@wso2/internal",
+    "plugin:@wso2/prettier",
```

Comment:
```suggestion
    "plugin:@wso2/prettier",
    "plugin:cypress/recommended"
```

Reply:
Create an issue in https://github.com/brionmario/wso2-ui-configs/issues repo to add a `@wso2/cypress` sharable config.

Reply:
https://github.com/brionmario/wso2-ui-configs/issues/8